### ISSUE TITLE
Request for Code Implementation of TENT and EATA Methods

### ISSUE THREAD

**Issue by heegwang27** [Hongbin98/MonoTTA#5]
Dear authors,

Thank you for your excellent work on MonoTTA. I was wondering if it would be possible to provide code implementations for the TENT and EATA methods proposed in your paper? I have been trying to implement the TENT and EATA methods but have encountered some difficulties in doing so. Having access to these implementations would be incredibly helpful for understanding and reproducing your results.

Looking forward to your response. Thank you in advance for your assistance!

**Comment by Hongbin98** [comment]
Hello @heegwang27, sorry for my late reply and thanks for your attention to our work~

I will provide my personal implementation of TENT and EATA this month and release their training code.

Meanwhile, could you provide a more detailed description of your difficulties? For instance, you can simply implement TENT by replacing the loss function part of our code with the entropy loss.
